crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -653,6 +655,11 @@ impl BatchProverLedgerOps for LedgerDB {
     ) -> anyhow::Result<Option<Vec<u32>>> {
         self.db.get::<CommitmentIndicesByL1>(&l1_height)
     }
+
+    #[instrument(level = "trace", skip(self))]
+    fn job_is_pending(&self, id: Uuid) -> bool {
+        self.db.get::<PendingL1SubmissionJobs>(&id).is_ok()
```

Comment:
i think this should be `(&id)?.is_some()`

Reply:
Also I am not sure if this is the "pending" that you intended. This table is populate when proving is finished, and job is waiting to be submitted to L1.

Reply:
There is no table that is populated before that. Those proving jobs are temporary, they don't count until they are proven.